tests/fv3_conf/coastal_hsofs_pahm2schism_run.IN

```diff
@@ -0,0 +1,13 @@
+mkdir RESTART
```

Comment:
@pvelissariou1 Why you have multiple copy script for PAHM related configuration. I think we could manage it to have single file. As we discussed in our last call, I think, we need to separate files related with SCHSIM vs. PAHM. Files that are related with SCHSIM and needs to be in the SCHSIM specific script and RT input directory. So, for example, the files under /work/noaa/nosofs/pvelissa/RT_takis/NEMSfv3gfs/input-data-20221101/SCHISM/Sandy can be move to SCHSIM directory and if there is a configuration file specific to PAHM component that could be handled by the `fv3_conf/coastal_pahm_run.IN` (we don't have one but we could create).